scripts/Dockerfile.verify

```
@@ -23,22 +23,24 @@ RUN apt-get update && \
 
-# Install Claude Code globally
-RUN npm install -g @anthropic-ai/claude-code@2
-
 # Copy entrypoint
 COPY --chmod=755 scripts/docker-entrypoint.sh /usr/local/bin/docker-entrypoint.sh
 
 USER testuser
 WORKDIR /home/testuser
 
+# Install Claude Code via native installer (WORKDIR must be set to avoid OOM from / traversal)
+RUN curl -fsSL https://claude.ai/install.sh | bash
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)